src/components/app/ProfileName.css

```diff
@@ -23,6 +23,11 @@
   white-space: nowrap;
 }
 
+.profileNameButton::before {
+  padding-right: 14px;
+  background: url(firefox-profiler-res/img/svg/edit-name-profiler.svg) no-repeat;
+}
```

Comment:
as you can see in other examples, if you use `background-image` with the URL this should work fine without the padding.

The CSS rule would merge with the existing one for `.menuButtonsButton-hasIcon::before` that you can see in `index.css`.

Have you tried that?

There could be problems due to how the merge happens (which one takes precedence over the other). If that happens we can look at how to fix this. Please try this simple solution and then we can look at it if this doesn't work properly.

Thanks!